backend/models/recipe.js

```diff
@@ -64,12 +63,24 @@ const RecipeSchema = new mongoose.Schema(
     urlSource: {
       type: String,
       required: false,
+      trim: false,
+    },
```

Comment:
I think image and cost can be removed from the schema since they are generated from api call